### PR TITLE
Add `Build from source` in `Installation options`

### LINKED ISSUE #4476: Add a Reference to `Build from source` from B.io
## Description
Currently, there is no information (no search results) in B.io about building Ballerina from the source, which is documented in [1].

Can add a link to it from the `Installation options` page.

[1] https://github.com/ballerina-platform/ballerina-distribution/blob/master/docs/building-from-source.md

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add `Build from source` in `Installation options`.
> Fixes #4476

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: learn/get-started/install-ballerina/Installation-options.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ Language specification 2022R1
 update tool 1.3.8
 ```
 
+## Build from source
+
+For instructions, see [Build from source](https://github.com/ballerina-platform/ballerina-distribution/blob/master/docs/build-from-source.md).
```
Comment (keizer619) [inline learn/get-started/install-ballerina/Installation-options.md:104]:
```suggestion
For instructions, see [Build from source](github.com/ballerina-platform/ballerina-distribution/blob/master/docs/building-from-source.md).
```

**Review by dakshitha — APPROVED** [review]

**Review by keizer619 — CHANGES_REQUESTED** [review]
url is not correct

File: learn/get-started/install-ballerina/Installation-options.md (hunk 1 above)
Comment (praneesha) [reply learn/get-started/install-ballerina/Installation-options.md:104]: This [file](https://github.com/ballerina-platform/ballerina-distribution/tree/master/docs) is renamed now via https://github.com/ballerina-platform/ballerina-distribution/pull/2880.

**Comment by praneesha** [comment]
> url is not correct

This [file](https://github.com/ballerina-platform/ballerina-distribution/tree/master/docs) is renamed now via https://github.com/ballerina-platform/ballerina-distribution/pull/2880.

**Pull request merged**